script/vsts/nightly-release.yml

```diff
@@ -47,9 +47,9 @@ jobs:
         env:
           GITHUB_TOKEN: $(GITHUB_TOKEN)
           ATOM_RELEASE_VERSION: $(ReleaseVersion)
-          ATOM_RELEASES_S3_KEY: $(ATOM_RELEASES_S3_KEY)
-          ATOM_RELEASES_S3_SECRET: $(ATOM_RELEASES_S3_SECRET)
-          ATOM_RELEASES_S3_BUCKET: $(ATOM_RELEASES_S3_BUCKET)
+          ATOM_RELEASES_S3_KEY: ${{ variables.ATOM_RELEASES_S3_KEY }}
+          ATOM_RELEASES_S3_SECRET: ${{ variables.ATOM_RELEASES_S3_SECRET }}
+          ATOM_RELEASES_S3_BUCKET: ${{ variables.ATOM_RELEASES_S3_BUCKET }}
           PACKAGE_CLOUD_API_KEY: $(PACKAGE_CLOUD_API_KEY)
```

Comment:
Also, I am not seeing the code skipping `PACKAGE_CLOUD_API_KEY`. Please do the same for that one too.